database/migrations/2021_04_06_113426_printcat_ora_setup.php

```diff
@@ -13,7 +13,7 @@ class PrintcatOraSetup extends Migration {
      */
     public function up() {
 
-        if (!Schema::hasTable('devices_faults_printers_ora_opinions')) {
+        if (!Schema::hasTable('devices_faults_tablets_ora_opinions')) {
```

Comment:
Not sure if this is from a different PR, but given this is the printcat migration, it should be checking for presence of `devices_faults_printers_ora_opinions`.

Reply:
It is.  I won't muddy things by fixing it in here.